src/NHibernate/Dialect/Oracle10gDialect.cs

```diff
@@ -29,19 +64,6 @@ protected override void RegisterFunctions()
 			RegisterFunction("random", new SQLFunctionTemplate(NHibernateUtil.Double, "cast(DBMS_RANDOM.VALUE() as binary_double)"));
 		}
 
-		/* 6.0 TODO: consider redefining float and double registrations
-		protected override void RegisterNumericTypeMappings()
```

Comment:
Here _useBinaryFloatingPointTypes is not yet populated as `RegisterNumericTypeMappings` is called in the constructor.